### DISCUSSION TITLE
Why does the training dataset only use 140 nodes？

### DISCUSSION THREAD

**Discussion by Chrisable** [pyg-team/pytorch_geometric#4593] [General]
![image](https://user-images.githubusercontent.com/49330390/166631730-a4515b72-aa7b-443c-9ef8-d382188cdd58.png)
When we use more nodes in the training phase, won't the accuracy be better? By the way, Can the mask be changed?

**Comment by rusty1s** [comment]
Yes, in general it holds that the model performance gets better in case you are using more training examples/nodes. You can change the mask to your needs via `transforms.RandomNodeSplit`.
